sirepo/package_data/static/js/ml.js

```diff
@@ -30,6 +30,14 @@ SIREPO.app.config(function() {
         <div data-ng-switch-when="TrimButton" class="col-sm-5">
           <div data-trim-button="" data-model-name="modelName" data-model="model" data-field="field"></div>
         </div>
+        <div data-ng-switch-when="URL" class="col-sm-5" data-field-class="fieldClass">
+          <input type="text" data-ng-model="model[field]" class="form-control" style="text-align: right" data-lpignore="true" />          
+          <span data-ng-show="model.dataOrigin === 'url'" style="font-style: italic; font-size: 80%;">{{ model.bytesLoaded || 0 }} of {{ model.contentLength || 0 }} bytes</span>
+          <div class="sr-input-warning"></div>
+          <div data-ng-show="model.contentLength && ! model.bytesLoaded" class="progress">
+            <div class="progress-bar progress-bar-striped active" role="progressbar" aria-valuenow="100" aria-valuemin="0" aria-valuemax="100" style="width:100%"></div>
+          </div>
+        </div>
```

Comment:
You can change the URL type handler class from "col-sm-5" to "col-sm-7" and it will show in a larger field. You can then remove the style="text-align: right" which was probably a work-around for the small field.

Reply:
[2683a56](https://github.com/radiasoft/sirepo/pull/4671/commits/2683a56139bbe105b349d17c92a723693e839d80)

I think the text align was from a copy-paste job